test/api.ts

```diff
@@ -3,22 +3,476 @@ import {assert} from 'chai'
 import {APIClient, FetchProvider} from '@wharfkit/antelope'
 import {mockFetch} from '@wharfkit/mock-data'
 
-import {ExampleAPI} from '$lib'
+import {AtomicAssetsAPIClient} from '$lib'
 
 // Setup an APIClient
 const client = new APIClient({
-    provider: new FetchProvider('https://jungle4.greymass.com', {fetch: mockFetch}),
+    provider: new FetchProvider('https://wax.api.atomicassets.io/', {fetch: mockFetch}),
 })
 
 // Setup the API
-const example = new ExampleAPI(client)
+const atomicassets = new AtomicAssetsAPIClient(client)
 
-suite('api', function () {
+suite('atomicassets', function () {
     this.slow(200)
     this.timeout(10 * 10000)
 
-    test('call test api', async function () {
-        const res = await example.get_info()
-        assert.equal(res.server_version, '905c5cc9')
+    test('get_config', async function () {
+        const res = await atomicassets.atomicassets.v1.get_config()
+        assert.equal(res.success, true)
```

Comment:
In general with these tests, I would probably check more than just the success boolean. At a minimum, I would make sure that the correct data type is being returned.

Reply:
Hmm, you are right, i originally thought typescript would ensure the response fullfills the responseTypes specified, but i may set totally wrong responseType.